projects/migrations/0003_auto_20151120_2121.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0002_create_project_model'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='project',
+            name='difficulty',
+            field=models.CharField(choices=[('beginner', 'beginner'), ('intermediate', 'intermediate'), ('advanced', 'advanced')], max_length=15),
+        ),
+    ]
```

Comment:
this migration is missing the actual change in the models. Usually is a good practice to create migrations (models change + migration file) as a single PR.